script/integration-test-rbac

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Instrument basic smoke RBAC test against minikube:
+# - _test_must_fail_without_rbac_roles for failure
+# - _test_must_pass_with_rbac_roles    for success
+#
+# Assumes already setup clster RBAC'd environment
+
+TEST_CONTEXT=${1:?missing k8s context, e.g.: minikube}
+KUBELESS_JSONNET=kubeless.jsonnet
+KUBELESS_JSONNET_RBAC=kubeless-rbac-novols.jsonnet
+
+DIR=${0%/*}
+source ${DIR}/libtest.sh || exit 1
+
+set -u
+## k8s specific Helper functions
+_wait_for_kubeless_controller_ready() {
+    info "Waiting for kubeless controller to be ready ... "
+    k8s_wait_for_pod_ready -n kubeless -l kubeless=controller
+    sleep 10
```

Comment:
New ThirdPartyResources take a while to become functional too.  I suggest (instead of the sleep) adding a loop that polls until `kubectl get functions` exits successfully - and see if that is sufficient.